### ISSUE TITLE
Field App: Viewing Previous Location Visit

### ISSUE THREAD

**Issue by mripp** [NWFWMD-IT/Hydrologic-Monitoring-Project#20]
In the Explorer application the user will be able to view previous location visits - what fields are needed to view?

Example:
- Previous Adjustment

**Comment by mmannion** [comment]
Test comment to notify @Bikechain

**Comment by Bikechain** [comment]
- Water Level
- Sensor Level
- Sensor Adjustment Amount
  - Only if a manual adjustment is entered (sensor adjusted = yes)
# Rain
- Rainfall bucket cleaned

**Comment by mmannion** [comment]
@mripp - @Bikechain and I were working on this tonight, started a list, and realized that someone may potentially need to see any one of the attributes from a previous visit.

In practice, there are some prior visit attributes that will be useful more / less frequently than others. But, field staff will typically be looking back at prior visits when something unusual is happening - and, by definition, we don't have a list of those "unusual" circumstances, else they'd be usual.

On the other hand, staff have been doing without most of this information for years. So, folks wouldn't be losing any functionality if we only include a handful of attributes from, say, the last month's worth of visits. But, it would be handy to have access to more data, if practical.

Let's talk about this topic more at the next team meeting. Questions are:

- What's the drawback to making every attribute from every prior visit available:
  - Space?
  - Performance?
  - Usability?

- If we do have to limit the information about prior visits, how many attributes / visits is it reasonable to include?

**Comment by mmannion** [comment]
Blocked on #108 
- Need to determine if we're using Explorer or Field Maps
  - Explorer can only navigate one relationship deep, which would require a view
  - We suspect Field Maps can navigate multiple relationships, which would allow taking the whole history offline in native format

- [ ] @mripp Research if Field Maps supports multi-level relationships

**Comment by mripp** [comment]
FieldMaps interface automatically allows for users to dive multiple layers deep on the screen. 
Ex: User clicks on a location, can view the associated measuring points, can view the previous site visits, can view stage measurements
Question: Does the user need to see all site visits or only the previous one?

ArcGIS Explore can only dive one level deep so a view would need to be created

**Comment by mmannion** [comment]
Using Field Maps instead of ArcGIS Explorer, so this is out-of-scope. See #112
